packages/host/app/resources/room.ts

```
@@ -396,7 +400,8 @@ export class RoomResource extends Resource<Args> {
     if (isCardInstance(skillCard)) {
       return skillCard;
     } else {
-      throw new Error(`Cannot find skill ${cardId}`);
+      // A known reason for this is that the skill has been renamed
+      return undefined;
```

Probably we should console log the issue, as opposed to silently doing nothing.